tests/login.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { expect, test } from '@playwright/test';
+
+/**
+ * This file contains tests that confirm our login page at /login works correctly.
+ */
+
+/**
+ * URLs used in this test file
+ */
+const LOGIN_PAGE = 'https://develop.shopcanal.com/login';
+const INVENTORY_PAGE = 'https://develop.shopcanal.com/shopkeep/inventory?page=1';
+
+/**
+ * Selectors used in this test file
+ */
+const EMAIL_INPUT_SELECTOR = 'input#email';
+const PASSWORD_INPUT_SELECTOR = 'input#password';
+const BUTTON_SELECTOR = 'button#login';
+
+/**
+ * Tests that the user can fill out the email and password inputs, then
+ * click login and be redirected to the app
+ */
+test('Login - can fill out valid email and password and successfully log in', async ({ page }) => {
+  await page.goto(LOGIN_PAGE);
+
+  // Fill out email and password
+  await page.fill(EMAIL_INPUT_SELECTOR, 'clay+canalshopkeep@shopcanal.com');
+  await page.fill(PASSWORD_INPUT_SELECTOR, 'cmsprodP@ssword1');
```

Comment:
This is a public repo, so need to remove this password from the commit. Force push to erase it from history as well! 

To provide passwords to tests, we'll need to look up what GitHub actions recommends to do